package.json

```diff
@@ -34,8 +34,8 @@
     "npmpub": "^3.1.0"
   },
   "scripts": {
-    "lint": "eslint .",
-    "tests": "ava '**/__tests__/*.js'",
+    "lint": "eslint ./src/",
```

Comment:
I would prefer this https://github.com/MoOx/statinamic/blob/2ab325c12d32b199e37488e1695313e35f9f1aea/package.json#L161